src/components/PaperUIKit/ErrorModal/ErrorModal.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Dialog, Paragraph, Text } from 'react-native-paper';
+import { ErrorType } from "../../../entities";
+import { Button, ButtonType } from "../Button";
+import styles from "./styles";
+
+
+interface ErrorModalProps {
+  error?: ErrorType,
+  action: () => void
+}
+
+const ErrorModal: React.FC<ErrorModalProps>  = ({ error, action = () => {} }) => {
+  if (!error) return null;
+  const { message } = error;
+  return (
+    <Dialog
+      visible
+      dismissable={false}>
+      <Dialog.Title>Error</Dialog.Title>
+      <Dialog.Content>
+        <Paragraph><Text style={styles.message}>{message}</Text></Paragraph>
+      </Dialog.Content>
+      <Dialog.Actions>
+        <Button title="OK" type={ButtonType.TEXT} onPress={action} />
```

Comment:
title="OK - how about translate?

Reply:
I think it's not necessary, because its normal when we use universal words (like Ok or Error) in components for showing errors. We can receive this value from props, but Im not sure about that. What do you think?